src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+Microsoft Azure CLI 'container' Command Module
+==============================================
+
+Commands to manage Azure container instances
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Group
+        az container: Manage Azure Container Instances.
+
+    Commands:
+        create : Create or update container group.
```

Comment:
This should only refer to creating a container group, and there should be a separate update command that uses the CLI's generic update template.